src/index.tsx

```diff
@@ -619,11 +657,9 @@ class App extends React.Component<{}, AppState> {
       >
         <fieldset>
           <legend>Shapes</legend>
-          {this.renderOption({ type: "rectangle", children: "Rectangle" })}
-          {this.renderOption({ type: "ellipse", children: "Ellipse" })}
-          {this.renderOption({ type: "arrow", children: "Arrow" })}
-          {this.renderOption({ type: "text", children: "Text" })}
-          {this.renderOption({ type: "selection", children: "Selection" })}
+          {SHAPES.map(shape =>
+            this.renderOption({ type: shape.value, children: shape.label })
```

Comment:
Can you get rid of this.renderOption function and inline it since we're not repeating it so many times now?

Reply:
You can use `forEach` instead of `map`

Reply:
In React you need to return a value (valid react element) inside `render` not just iterate over an array

Reply:
Fixed https://github.com/excalidraw/excalidraw/pull/85/commits/19d5c7e73017b948b5a9e5f8e1fee6a29d90c602